README.rst

```diff
@@ -12,9 +12,9 @@ The repository includes a set of branches to maintain different stack versions d
 
   Examples:
 
-  - ``branch-1.0`` - Python 2 branch with Scrapy 1.0
-  - ``branch-1.1`` - Python 2 branch with Scrapy 1.1
-  - ``branch-1.1-py3`` - Python 3 branch with Scrapy 1.1
+  - ``branch-1.0`` - Branch with Scrapy 1.0
+  - ``branch-1.1`` - Branch with Scrapy 1.1
+  - ``branch-1.1-py3`` - Branch with Scrapy 1.1, using the Python 3
```

Comment:
I think this was more clear before. Maybe you could simply add a 4th line for branch-2.0 which indicates it uses Python 3, for clarity.

Reply:
That was by design. The illustration that 2.0 and later already users python 3 is given just below, the Note section.

Still, I can add if you want.

Reply:
I don’t mind much whether or not you add that. But I do think that this change should not remove the explicit mention of Python 2 from before. I think that was useful information.

Reply:
OK, reverted.